dagger-functions/src/test/java/io/odpf/dagger/functions/transformers/FeatureTransformerTest.java

```diff
@@ -24,6 +25,9 @@
     @Mock
     private Configuration configuration;
 
+    public FeatureTransformerTest() {
```

Comment:
what is this for?

Reply:
Missed this. Not being used anywhere. Removing this.